src/components/spaces/proteins/protein-viewer.sass

```diff
@@ -1,23 +1,47 @@
 .protein-viewer
   display: flex
-  align-items: center
   position: initial !important
 
 .proteins
   display: flex
 
+  .protein
+    display: flex
+    padding: 0 10px
+
+  .protein > svg
+    height: 40vh
+
 .amino-acids
   position: absolute
   bottom: 5px
 
-.options
-  position: absolute
-  bottom: 5px
-  display: flex
-  flex-direction: column
-  align-items: flex-start
+.info-and-options
+  font-size: 0.8em;
```

Comment:
I know we bounce between em and px - at some point we should try to unify these.  Not urgent, just hoping that typing this will make it stick in my head for future reference.